src/selectors/nodes.js

```diff
@@ -113,6 +113,15 @@ export const getNodeLabel = createSelector(
     isPrettyName ? nodeName : nodeFullName
 );
 
+/**
```

Comment:
Can we use getNodeName and getNodeFullName selectors for searching instead of creating a new selector ?

Reply:
I think we have 2 selectors at line 13. 14 in selectors -> nodes.js 

```
export const getNodeName = (state) => state.node.name;
export const getNodeFullName = (state) => state.node.fullName)
```
These do not depend on pretty flag. Can you please check once.